src/core/positioning/positioning.h

```diff
@@ -67,6 +67,10 @@ class Positioning : public QObject
 
     Q_PROPERTY( bool backgroundMode READ backgroundMode WRITE setBackgroundMode NOTIFY backgroundModeChanged )
 
+    Q_PROPERTY( bool averagePositionFilterAccuracy READ averagePositionFilterAccuracy WRITE setAveragePositionFilterAccuracy NOTIFY averagePositionFilterAccuracyChanged )
```

Comment:
average -> averaged (to match the two other properties we have above. Also, I'd relocate that property to see with the two averaged* properties.

```suggestion
    Q_PROPERTY( bool averagedPositionFilterAccuracy READ averagedPositionFilterAccuracy WRITE setAveragedPositionFilterAccuracy NOTIFY averagedPositionFilterAccuracyChanged )
```